api/README.md

```diff
@@ -14,7 +14,20 @@ These flask apps are deployed as long-running tasks in ECS and are reverse proxy
 
 ## Local usage
 
```

Comment:
nbd but i would prefer if the config and secrets sections lived above local usage, first things first in a new app directory i wanna get the env secrets sorted out :shrug: